File: Axiom/Internal/Ast/SequenceExpression.cs
using System.Collections.Generic;

namespace Axiom.Internal.Ast
{
    internal class SequenceExpression : Expression
    {
        public SequenceExpression(IList<Expression> expressions)
        {
            Expressions = expressions;
        }

        public override void Accept(IVisitor visitor)
        {
            visitor.Visit(this);
        }

        public IList<Expression> Expressions { get; private set; }
    }
}

File: src/main/java/com/moybl/axiom/ast/SequenceExpression.java
package com.moybl.axiom.ast;

import java.util.List;

public class SequenceExpression implements Expression {

	private List<Expression> expressions;

	public SequenceExpression(List<Expression> expressions) {
		this.expressions = expressions;
	}

	public void accept(Visitor visitor) {
		visitor.visit(this);
	}

	public List<Expression> getExpressions() {
		return expressions;
	}

}
